### DISCUSSION TITLE
any proxies i can use that arent herokuapp.... heroku isnt free any more

### DISCUSSION THREAD

**Discussion by Theblackmamba824** [3kh0/ext-remover#297] [Help JOIN TITANIUM NETWORK TO SHARE PLEASE] [answered]
any 1 who have links to proxies they can use pls put them in

**Comment by rwg100** [comment, marked as answer]
[https://womginx1.lunarvcoding.repl.co/ ](url)

[https://bananaboat.me/ ](url)


clicking the link dont work, you have to copy and paste the link into the url bar

they are both managed by NGINX
